src/js/personalization/va-profile-beta/utils/address.js

```diff
@@ -38,7 +38,7 @@ const ADDRESS_TYPES = {
  * @param {Address} address
  * @returns {Address}
  */
-function consolidateAddress(address) {
+export function consolidateAddress(address) {
```

Comment:
For 'US' country code country name, I think 'United States' might be preferred, @markgreenburg pointed this out previously.